classes/class-pmpro-subscription.php

```diff
@@ -148,6 +148,38 @@ public function __construct( $subscription = null ) {
 		}
 	}
 
+	/**
+	 * Get a property of this object.
+	 *
+	 * @since TBD
+	 *
+	 * @param  string $property The property to get.
+	 * @return mixed  The property value.
+	 */
+	public function __get( $property ) {
+		if ( isset( $this->{$property} ) ) {
+			return $this->{$property};
+		}
+	}
+
+	/**
+	 * Set a property of this object.
+	 *
+	 * @since TBD
+	 *
+	 * @param  string $property The property to set.
+	 * @param  mixed  $value    The value to set.
+	 * @return bool   True if the property was set, false if not.
+	 */
+	public function __set( $property, $value ) {
+		// TODO: Validate values if needed.
+		if ( isset( $this->{$property} ) ) {
+			$this->{$property} = $value;
+			return true;
+		}
+		return false;
+	}
+
 	/**
```

Comment:
We probably should add an `__isset( $name )` method too to deal with potential empty() usage in the future by other developers.